scripts/utils/deleteServerlessService.js

```diff
@@ -0,0 +1,43 @@
+const { spawn } = require('child_process');
+const fs = require('fs');
+const path = require('path');
+
+// Get config
+const config = require(`../../config.json`); // eslint-disable-line import/no-unresolved
+
+// Helpers
+const resolveSymbolicLink = filePath => {
+  const lstat = fs.lstatSync(filePath);
+  return lstat.isSymbolicLink()
+    ? path.resolve(path.dirname(filePath), fs.readlinkSync(filePath))
+    : false;
+};
+
+module.exports = service => {
+  console.log(`Start deletion of ${service}`);
+  console.time(service);
+
+  const operation = spawn(
+    `./node_modules/.bin/sls`,
+    ['remove', `--stage ${config.stage}`, `--region ${config.region}`],
+    {
+      cwd: resolveSymbolicLink(`node_modules/@eubfr/${service}`),
+      // This is a helper for serverless.yml variables as an alternative to options
+      env: {
+        EUBFR_USERNAME: process.env.EUBFR_USERNAME
```

Comment:
`EUBFR_USERNAME: process.env.EUBFR_USERNAME || ''` is shorter

Reply:
As reported on slack, it only works in my environment if I remove this line. (`env: { ...`)